src/subdomains/supporting/bank/fiat-output/dto/update-fiat-output.dto.ts

```diff
@@ -0,0 +1,124 @@
+import { IsBoolean, IsDate, IsNumber, IsOptional, IsString } from 'class-validator';
+import { TransactionCharge } from '../../bank-tx/frick.service';
+
+export class UpdateFiatOutputDto {
+  @IsOptional()
+  @IsNumber()
+  typeId?: number;
```

Comment:
`typeId` is not really a fitting name, sounds like id of the type 😁 maybe `originEntityId` or something?